app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def file_creator():
+    name_of_file = input("Enter name of file:")
```

Comment:
Enter name of the file: name1
Enter new line of content: This is the first line of content

You are forget about space after Enter name of the file: